README.md

```diff
@@ -95,10 +95,10 @@ https://forms.gle/u1RXnbocbFWqfxGb9
 
 ```bash
 # run the following command in 1 terminal
-docker-compose up -f docker-compose.dev.yml --force-recreate --build
+docker-compose --file ./docker-compose.dev.yml up --force-recreate --build
```

Comment:
Using docker-compose 1.24.1, this is the only syntax that would successfully build.